src/META-INF/plugin.xml

```diff
@@ -48,11 +48,11 @@
             </tr>
             <tr>
               <td align="right"><b>Support:</b></td>
-              <td align="left"><a style="color:white;" href="http://community.haxe.org/">Community Discourse</a></td>
+              <td align="left"><a style="color:white;" href="https://community.haxe.org/">Community Discourse</a></td>
             </tr>
             <tr>
               <td align="right"><b>Enterprise Support:</b></td>
-              <td align="left"><a style="color:white;" href="http://bishtonsoftwaresolutions.com/">Bishton Software Solutions</a></td>
+              <td align="left"><a style="color:white;" href="https://bishtonsoftwaresolutions.com/">Bishton Software Solutions</a></td>
             </tr>
             <tr>
```

Comment:
The same is true for this stanza, since I no longer am accepting donations via Patreon.

Reply:
Also, the table at (62-66)